examples/alloy-editor/app.js

```diff
@@ -1,11 +1,14 @@
-import widget from '../../lib/api'
+import { init } from '../../lib/api'
+
+const AlloyEditor = window.AlloyEditor
+
+init((api) => {
```

Comment:
Let’s call the parameter `widget` to be consistent with the rest of the examples and docs